examples/jsm/controls/OrbitControls.js

```diff
@@ -21,9 +20,9 @@ import {
 const _changeEvent = { type: 'change' };
 const _startEvent = { type: 'start' };
 const _endEvent = { type: 'end' };
-const _ray = new Ray();
-const _plane = new Plane();
-const TILT_LIMIT = Math.cos( 70 * MathUtils.DEG2RAD );
+const _ray = /* @__PURE__ */ new Ray();
+const _plane = /* @__PURE__ */ new Plane();
+const TILT_LIMIT = Math.cos( 70 * ( Math.PI / 180 ) );
```

Comment:
Accessing `MathUtils` seems to break tree-shaking. Noticed this in `three-stdlib` also. I intend to revert this.